packages/main/src/Input.js

```diff
@@ -526,6 +569,28 @@ class Input extends UI5Element {
 		}
 	}
 
+	toggle(bToggle) {
```

Comment:
we don't use hungarian notation, just overwrite the parameter on the next line, no need for another variable

Reply:
Done.